src/routes/auth/login.ts

```diff
@@ -76,7 +80,7 @@ async function loginAccount({ body }: Request, res: Response): Promise<unknown>
     throw Boom.badRequest('Account is not activated.')
   }
 
-  if (!(await bcrypt.compare(password, password_hash))) {
+  if (!(await bcrypt.compare(password, password_hash)) && req.header(AdminSecretHeader) != ADMIN_SECRET) {
```

Comment:
should probably be `||` instead of `&&` here.

Reply:
At first i thought you might be right but the tests broke after this change.
And it make sense:

The && in that case stands for:
"If neither the password and the header is right, send back the error"
A || would stands for :
"If the password or the header is not right, send back the error"
Which will not work if a client does not send the AdminSecret header